tox.ini

```diff
@@ -1,15 +1,16 @@
 [tox]
 envlist = py35-django22,py38-django{22,30}, quality
 
+[pytest]
+addopts = --cov=tests --cov-report term --cov-config=.coveragerc -p no:randomly
+testpaths = tests
+
 [testenv]
 deps =
     django22: Django>=2.2,<2.3
     django30: Django>=3.0,<3.1
     -r requirements/test.txt
-
-commands =
-    coverage run -m nose
-    coverage report
+commands = {posargs:pytest}
```

Comment:
I'm wondering what sort of syntax is this? should not it be `commands = pytest {posargs}`?
https://tox.readthedocs.io/en/latest/example/general.html